testcases/HelloWorld.py

```diff
@@ -18,17 +18,26 @@
 # permissions and limitations under the License.
 
 import unittest
+import logging
 
 import OpTestConfiguration
+import OpTestLogger
 from common.OpTestUtil import OpTestUtil
 from common.OpTestSystem import OpSystemState
 
+module_logger = logging.getLogger(OpTestLogger.optest_logger_glob.parent_logger+'.{}'.format(__name__))
```

Comment:
Can we have a helper in OpTestLogger itself, for getting new instances of logger with input as module name `__name__`.
So that we can reuse that in all testcases like this.
```
self.logger =  OpTestLogger.get_logger(__name__)
```

Reply:
Done